src/scripts.js

```diff
@@ -112,7 +111,7 @@ function printPastTrips() {
   filteredTrips.innerHTML = ``
   pastTrips.forEach(trip => {
     let destination = destinationsRepo.findDestination(trip.destinationID);
-    filteredTrips.innerHTML += `<div id="${trip.id}"><h4>${destination.destination}<h4><p>${trip.date}</p></div>`;
+    filteredTrips.innerHTML += `<div id="${trip.id}"><h3>${destination.destination}<h3><p>${trip.date}</p></div>`;
```

Comment:
Swear I had an h3 hiding in here before...oh right I remember where now it was the second hero section for a secondary trip.